### PR TITLE
Made a programm to test if a number is a Triangular Number and also made tests

### PR DESCRIPTION
**Pull request by RF-Buerky**
One solution for issue #26

### CODE REVIEW

**Comment by RF-Buerky** [comment]
As always:
The test failure has occured because all the tests are executed. So the tests in
* Running org.codefx.demo.bingen.energy_model.AccountTest
* Running org.codefx.demo.bingen.energy_model.math.RomanNumeralsTest

failed.

However the given pull request has a working version for `TriangularNumber.kt` and `TriangularNumberTest`.

View the tests log lines 960 and 961 to get the proof. There you can read:
`Running org.codefx.demo.bingen.energy_model.math.TriangularNumberTest`
`Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec`

File: src/test/kotlin/org/codefx/demo/bingen/energy_model/math/TriangularNumberTest.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class TriangularNumberTest {
+
+    @Test
+    fun isTriangularNumber10() {
+        val test = isTriangularNumber(10)
+        assertEquals(true, test)
+    }
+
+    @Test
+    fun isTriangularNumberMinus10() {
```
Comment (RF-Buerky) [inline src/test/kotlin/org/codefx/demo/bingen/energy_model/math/TriangularNumberTest.kt]: I first wanted to name this funktion `isTriangularNumber-10` but the minus made some syntax mistakes occur. Is there a way to use things like this `-` in the functions name?

Comment (RF-Buerky) [inline src/test/kotlin/org/codefx/demo/bingen/energy_model/math/TriangularNumberTest.kt]: I first wanted to name this funktion `isTriangularNumber-10` but the minus made some syntax mistakes occur. Is there a way to use things like this `-` in the functions name?

Comment (nicolaiparlog) [inline src/test/kotlin/org/codefx/demo/bingen/energy_model/math/TriangularNumberTest.kt]: Nope, only letters and digits.

**Pull request merged**